app/controllers/application_controller.rb

```diff
@@ -1,10 +1,142 @@
-require './config/environment'
-
 class ApplicationController < Sinatra::Base
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller).